src/codeflare_sdk/ray/cluster/build_ray_cluster.py

```diff
@@ -243,14 +252,23 @@ def update_image(image) -> str:
     return image
 
 
-def get_pod_spec(cluster: "codeflare_sdk.ray.cluster.Cluster", containers):
+def get_pod_spec(cluster: "codeflare_sdk.ray.cluster.Cluster", containers, tolerations):
     """
     The get_pod_spec() function generates a V1PodSpec for the head/worker containers
     """
-    pod_spec = V1PodSpec(
-        containers=containers,
-        volumes=generate_custom_storage(cluster.config.volumes, VOLUMES),
-    )
+
+    if tolerations is None:
+        pod_spec = V1PodSpec(
+            containers=containers,
+            volumes=generate_custom_storage(cluster.config.volumes, VOLUMES),
+        )
+    else:
+        pod_spec = V1PodSpec(
+            containers=containers,
+            volumes=generate_custom_storage(cluster.config.volumes, VOLUMES),
+            tolerations=tolerations,
+        )
```

Comment:
This can be reduced to:
```suggestion
    pod_spec = V1PodSpec(
        containers=containers,
        volumes=generate_custom_storage(cluster.config.volumes, VOLUMES),
        tolerations=tolerations or None,
    )
```

Reply:
@ChristianZaccaria Thank you for improvement. :+1: